layers/+spacemacs/spacemacs-layouts/funcs.el

```diff
@@ -55,6 +55,25 @@ current perspective."
   (helm-mini)
   (add-hook 'ido-make-buffer-list-hook  #'persp-restrict-ido-buffers))
 
+(defun spacemacs/find-file-in-project (filename)
+  "Open a file like find-file. If the file belongs to a project, creates
+   a new persp and enables projectile mode for it."
+  (interactive "P")
```

Comment:
Doesn't `(interactive "P")` mean that the first function argument will be the numeric prefix argument?